src/os/osexec.go

```diff
@@ -0,0 +1,55 @@
+//go:build linux && !baremetal && !darwin && !tinygo.wasm
+
+package os
+
+import (
+	"errors"
+	"syscall"
+	"unsafe"
+)
+
+func fork() (pid int, err error) {
+	// ret := libc_fork()
+	// ret, _, _ := syscall.Syscall(syscall.SYS_FORK, 0, 0, 0)
+	ret, _, _ := syscall.Syscall(57, 0, 0, 0)
```

Comment:
Don't use magic numbers. Use `syscall.SYS_FORK` instead, for example.

Reply:
I removed the explicit syscall number and found out, that in the CI the definition for SYS_FORK cannot be found. Grepping through the source tree I found out, that there is no definition for this, only the local go installation provides this, e.g. via src/syscall/zsysnum_linux_amd64.go. What confused me is that the CI should have picked this up but did not. On my local machine, this works perfectly fine.